### ISSUE TITLE
Copybara should follow the XDG Base Directory Specification

### ISSUE THREAD

**Issue by sudoforge** [google/copybara#151]
The [XDG Base Directory Specification][0] defines base directories in which specified files should live. Of these, I think `XDG_DATA_HOME` (or `~/.local/share`) makes the most sense for Copybara. My main motivation for this is to avoid the top-level `copybara` directory in my `$HOME`.

[0]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

**Comment by rpavlik** [comment]
Yes, this would be really good. I would imagine there's probably a java package for this purpose.

**Comment by sudoforge** [comment]
@rpavlik i actually submitted a PR for this: https://github.com/google/copybara/pull/152

Unfortunately, @mikelalcon [mentioned](https://github.com/google/copybara/pull/152#issuecomment-742674079) that several internal processes/tools depend on the current default output root. As such, I simply set an alias in my shell's RC file. You may benefit from doing the same:

https://github.com/sudoforge/dotfiles/blob/trunk/zsh/.zshrc#L205
